src/http.zig

```diff
@@ -3158,7 +3161,8 @@ fn handleResponseBodyFromSinglePacket(this: *HTTPClient, incoming_data: []const
     defer {
         if (this.progress_node) |progress| {
             progress.activate();
-            progress.setCompletedItems(incoming_data.len);
+            this.downloaded_bytes += incoming_data.len;
+            progress.setName(std.fmt.allocPrint(std.heap.page_allocator, "Downloading [{s:.2}/{s:.2}]", .{ bun.fmt.size(this.downloaded_bytes), bun.fmt.size(this.estimated_content_length) }) catch bun.outOfMemory());
```

Comment:
this memory is never freed. an allocation is not really needed for this use case, and it definitely should not re-allocate the message per frame. consider using a stack-owned buffer as described in [this comment](https://github.com/oven-sh/bun/pull/11402#pullrequestreview-2083549656)